cliv2/Makefile

```
@@ -149,7 +154,8 @@ configure: summary $(CACHE_DIR) $(CACHE_DIR)/variables.mk $(V1_DIRECTORY)/$(V1_E
 
 $(BUILD_DIR)/$(V2_EXECUTABLE_NAME): $(BUILD_DIR) $(SRCS) generate-ls-protocol-metadata
 	@echo "$(LOG_PREFIX) Building ( $(BUILD_DIR)/$(V2_EXECUTABLE_NAME) )"
-	@GOEXPERIMENT=$(FIPS_CRYPTO_BACKEND) GOOS=$(_GO_OS) GOARCH=$(GOARCH) $(GOCMD) build -tags=application -ldflags="$(LDFLAGS) -X github.com/snyk/snyk-ls/application/config.Version=$(LS_COMMIT_HASH) -X github.com/snyk/snyk-ls/application/config.LsProtocolVersion=$(LS_PROTOCOL_VERSION)  -X main.internalOS=$(GOOS) -X github.com/snyk/cli/cliv2/internal/embedded/cliv1.snykCLIVersion=$(CLI_V1_VERSION_TAG)" -o $(BUILD_DIR)/$(V2_EXECUTABLE_NAME) $(WORKING_DIR)/cmd/cliv2/*.go
+	@if [ "$(DEBUG)" = "1" ]; then echo "DEBUG is set to 1 LDFLAGS=$(LDFLAGS) GCFLAGS=$(GCFLAGS)"; fi
```

nitpick: Just to adapt the log output to the rest of the log messages.

```suggestion
	@echo "$(LOG_PREFIX)  DEBUG Build: $(DEBUG)"
	@echo "$(LOG_PREFIX)  LDFLAGS:     $(LDFLAGS)"
	@echo "$(LOG_PREFIX)  GCFLAGS:     $(GCFLAGS)"
```